libs/ui-lib/lib/ocm/components/clusterConfiguration/platformIntegration/ExternalPlatformDropdown.tsx

```diff
@@ -8,35 +8,46 @@ import {
   FormGroup,
   Split,
   SplitItem,
+  Tooltip,
 } from '@patternfly/react-core';
 import { CaretDownIcon } from '@patternfly/react-icons';
 import { useField } from 'formik';
-import { getFieldId } from '../../../../common';
+import { FeatureId, getFieldId } from '../../../../common';
 import {
+  ExternalPlaformIds,
   ExternalPlatformLabels,
   ExternalPlatformLinks,
   ExternalPlatformTooltips,
 } from './constants';
 import { PlatformType } from '@openshift-assisted/types/assisted-installer-service';
+import {
+  NewFeatureSupportLevelData,
+  NewFeatureSupportLevelMap,
+  useNewFeatureSupportLevel,
+} from '../../../../common/components/newFeatureSupportLevels';
 
 const INPUT_NAME = 'platform';
 const fieldId = getFieldId(INPUT_NAME, 'input');
 
 type ExternalPlatformDropdownProps = {
   showOciOption: boolean;
-  disabledOciTooltipContent: React.ReactNode;
-  isOciDisabled: boolean;
   onChange: (selectedPlatform: PlatformType) => void;
+  cpuArchitecture?: string;
+  featureSupportLevelData?: NewFeatureSupportLevelMap | null;
```

Comment:
It is already `null` when the data is not yet available... Why is it also optional?
IMO this is making the code harder to interpret by the reader. Please try to unify the cases where the data may not be ready, make it either `null` or `undefined`. Don't make the both cases indicate the fact that the data is not ready.